retraites/SimulateurRetraites.py

```diff
@@ -33,6 +33,10 @@ def __init__(self, json_filename = None):
         8 pilotageParAgeCotisationsDepenses
         9 pilotageParAgeEtNiveauDeVie (sous-entendu et par solde financier)
         10 pilotageParNiveauDeVieEtCotisations (sous-entendu et par solde financier)
+        
+        Les indices de scénarios vont de 1 à 6, dans l'attribut "scenarios"
```

Comment:
Suggestion sur plusieurs lignes. Lire cette suggestion conjointement avec les 2 suivantes.
```suggestion
        Les scénarios sont numérotés de 1 à 6 dans l'attribut "scenarios"
```